src/allencell_ml_segmenter/_tests/prediction/test_model_input_widget.py

```diff
@@ -116,53 +87,12 @@ def test_postprocessing_method(
     with qtbot.waitSignal(model_input_widget._top_postproc_button.toggled):
         model_input_widget._top_postproc_button.click()
 
-    # ASSERT
-    assert (
-        model_input_widget._model.get_postprocessing_method()
-        == ModelInputWidget.TOP_TEXT
-    )
-
     # ACT
     with qtbot.waitSignal(model_input_widget._mid_postproc_button.toggled):
         model_input_widget._mid_postproc_button.click()
 
     # ASSERT
-    assert (
-        model_input_widget._model.get_postprocessing_method()
-        == ModelInputWidget.MID_TEXT
-    )
 
     # ACT
     with qtbot.waitSignal(model_input_widget._bottom_postproc_button.toggled):
         model_input_widget._bottom_postproc_button.click()
-
-    # ASSERT
-    assert (
-        model_input_widget._model.get_postprocessing_method()
```

Comment:
are all the deletes here covering dead code?

Reply:
this is all dead code